examples/listbox/listbox-collapsible.html

```diff
@@ -304,8 +304,10 @@ <h2 id="sc_label">HTML Source Code</h2>
     <div id="sc_start_sep" role="separator" aria-labelledby="sc_start_sep sc_label" aria-label="Start of "></div>
     <pre><code id="sc"></code></pre>
     <div id="sc_end_sep" role="separator" aria-labelledby="sc_end_sep sc_label" aria-label="End of"></div>
-    <script>sourceCode.add('sc', 'ex', 'ex_label', 'css_js_files')</script>
-    <script> sourceCode.make() </script>
+    <script>sourceCode.add('sc', 'ex', 'ex_label', 'css_js_files');
+</script>
+    <script>sourceCode.make();
+ </script>
```

Comment:
Can you fix this formatting? (Also in other files.)

Reply:
Unfortunatly, that is how it auto-fixes it. I reverted some of them, but then they change back after

Reply:
What happens if you include a newline after `<script>`?

Reply:
Think it's a known issue https://github.com/BenoitZugmeyer/eslint-plugin-html/issues/131

Reply:
In files where there is a newline after the start tag the indentation is OK. Can you add a newline after the start tag here and see what autofix does?
```suggestion
    <script>
      sourceCode.add('sc', 'ex', 'ex_label', 'css_js_files');
    </script>
    <script>
      sourceCode.make();
    </script>
```